### DISCUSSION TITLE
The current status of the Industry sessions still showing incomplete

### DISCUSSION THREAD

**Discussion by trishna-123** [github/india#205] [Students]
I have watched all the videos of Industry session but then also status is showing that they're incomplete. Do reply to this, I have asked before also but reply from your side. Also mailed the team but no response. SO, kindly reply to this.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
